### PR TITLE
[11.0] [MIG] hr_timesheet_report

### PR DESCRIPTION
**Pull request by Aldeigja**
Backport to 11.0

### CODE REVIEW

**Review by geomer198 — APPROVED** [review]
Code Review ok!

**Review by ivs-cetmix — APPROVED** [review]
Functional review OK

**Comment by Aldeigja** [comment]
@OCA/human-resources-maintainers I realize this is an old version, but could you please take a look?

File: hr_timesheet_report/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 {
     'name': 'Task Logs Timesheet Report',
-    'version': '12.0.1.0.1',
+    'version': '11.0.1.0.1',
```
Comment (alexey-pelykh) [inline hr_timesheet_report/__manifest__.py:7]: `11.0.1.0.0`, reset the version

File: hr_timesheet_report/report/hr_timesheet_report.py [hunk 2]
```diff
@@ -689,23 +689,24 @@ def _render_value_cell(self, row, col, sheet, formats, entry, field):
...
-            sheet.write_datetime(
+            sheet.write(
                 row,
                 col,
                 raw_value,
                 formats['cell_datetime']
             )
         elif field.field_type == 'date':
-            sheet.write_datetime(
+            sheet.write(
```
Comment (alexey-pelykh) [inline hr_timesheet_report/report/hr_timesheet_report.py:699]: Why?

Comment (alexey-pelykh) [reply hr_timesheet_report/report/hr_timesheet_report.py:699]: If the underlying type changed to string, parse it to date and write date

**Pull request closed without merging**